packages/playground/src/createDefaultUglyUploadUI.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  StoreOptionsShowUploadUI,
+  Signal,
+  Disposable,
+  flattenDisposable,
+} from '@blocksuite/store';
+import { renderSpec } from './renderSpec';
+import { ChangeEvent } from 'react';
+
+export function createDefaultUglyUploadUI(options: {
+  mountContainer: HTMLElement;
+}): StoreOptionsShowUploadUI {
+  const $promptText$ = new Signal<string>();
+  const $filesSelected$ = new Signal<FileList | null>();
+  const $show$ = new Signal<boolean>();
+  const $cancel$ = new Signal<true>();
+  const { dom } = renderSpec(document, [
```

Comment:
I don' really understand why do we need `renderSpec` pattern here, if we are just adding it for mocking example UI, this looks overkill.